R/dataProcessPlots.R

```diff
@@ -303,36 +392,45 @@ dataProcessPlots = function(
         quant$censored = FALSE
       }
       quant$analysis = "Run summary"
+      quant$newABUNDANCE = quant$ABUNDANCE
       single_protein$analysis = "Processed feature-level data"
+      print(colnames(single_protein))
       combined = rbind(single_protein[
         , 
         list(PROTEIN, PEPTIDE, TRANSITION, FEATURE, LABEL,
-             RUN, ABUNDANCE, FRACTION, censored, analysis)], quant)
+             RUN, ABUNDANCE, newABUNDANCE,FRACTION, censored, analysis)], quant,fill=TRUE)
       combined$analysis = factor(combined$analysis)
       combined$FEATURE = factor(combined$FEATURE)
       combined$RUN = as.numeric(combined$RUN)
+      print(colnames(combined))
       profile_plot = .makeSummaryProfilePlot(
         combined, is_censored, y.limdown, y.limup, x.axis.size, y.axis.size, 
         text.size, text.angle, legend.size, dot.size.profile, cumGroupAxis, 
         yaxis.name, lineNameAxis, groupNametemp
       )
       print(profile_plot)
       setTxtProgressBar(pb, i)
+      output_plots[["summary_plot"]][[paste("plot",i)]] <- profile_plot
+      
     }
     close(pb)
     
-    if (address != FALSE) {
+    if (address != FALSE & isPlotly == FALSE) {
       dev.off()
     } 
   }
+  if(isPlotly) {
+      return(output_plots)
```

Comment:
we don't use return, it's OK to just do
if (isPlotly) {
  output_plots
}
or change value of output_plots depending on other params and always return it

Reply:
Commit [remove return statements](https://github.com/Vitek-Lab/MSstats/pull/107/commits/f9695cb099b2ff0f094132d00932ced607fd91bd)